.buildkite/scripts/build-docs.sh

```diff
@@ -13,6 +13,7 @@ export GIT_COMMITTER_EMAIL=$GIT_AUTHOR_EMAIL
 build_args=""
 rebuild_opt=""
 skiplinkcheck_opt=""
+warnlinkcheck_opt=""
```

Comment:
let's streamline a bit and use a single variable for the broken_links_opt that will either have the `--skiplinkcheck` or `--warnlinkcheck` value

Reply:
fixed